_i18n/en/documentation/general/ads.md

```diff
@@ -1,4 +1,19 @@
-The official versions of AntennaPod are completely ad-free and open source! If you are seeing ads, this can have two reasons:
+The official versions of AntennaPod **do not insert any ads**. You can even verify this because [AntennaPod's source code is public](https://github.com/AntennaPod/AntennaPod). Apart from that, AntennaPod is developed by volunteers without commercial interest.
```

Comment:
```suggestion
The official versions of AntennaPod do not insert any ads. The app is developed by volunteers without commercial interest and the project relies on donations to cover costs. As AntennaPod's [source code](https://github.com/AntennaPod/AntennaPod) and [income and expenses](https://opencollective.com/antennapod#category-BUDGET) are public everyone is free to verify this.
```
I can't verify "can" not verify this in code code because I don't have the skills - hence 'free to'.